Components/buttons.css

```diff
@@ -0,0 +1,247 @@
+
+.btn{
+    display:inline-block;
+    font-size: 1rem;
+    padding:0.5rem 1rem;
+    margin:1.25rem 0.4rem;
+    background-color: transparent;
+    cursor:pointer;
+    border-radius:0.25rem;
+    border: none;
+    outline: none;
+    text-align: center; 
+
+}
+/* Primary Button styles */
+.primary-btn{
+    color:var(--complementary-text-color);
+    background-color: var(--primary-color);
+}
+
+.primary-btn:hover{
+    background-color: var(--dark-primary-color);
+}
+
+.primary-btn-outline{
+    color:var(--primary-color);
+ border:1px solid var(--primary-color);
```

Comment:
It is better to use prettier to format the code for consistent look

Reply:
used the prettier extension